.github/workflows/build.yml

```diff
@@ -128,6 +128,37 @@ jobs:
         name: reactos-syms-msvc${{matrix.toolset}}-${{matrix.arch}}-${{github.sha}}
         path: build/msvc_pdb
 
+  build-msvc-arm:
+    strategy:
+      matrix:
+        arch: [arm]
+        toolset: ['14.2', '14.1'] # VS 2019, 2017
```

Comment:
Add a comment why not `14.0` too? Especially as you wrote 'Tested VS 2015 arm ninja build (rapps builds)'.